packages/contracts-bedrock/contracts/L2/GasPriceOracle.sol

```diff
@@ -16,7 +15,14 @@ import { L1Block } from "../L2/L1Block.sol";
  *         contract exposes an API that is useful for knowing how large the L1 portion of their
  *         transaction fee will be.
  */
-contract GasPriceOracle is Ownable, Semver {
+contract GasPriceOracle is Semver {
+    /**
+     * @custom:legacy
+     * @custom:spacer _owner
+     * @notice Spacer for backwards compatibility.
+     */
+    address private spacer_0_0_32;
```

Comment:
Should be 0_0_20 to be exact

Reply:
ah good catch

Reply:
forgot to update the name when i updated the type